Sources/SwiftSyntaxBuilder/Buildables.swift.gyb

```diff
@@ -80,6 +102,11 @@ extension ${kind}Buildable {
 
 % for node in SYNTAX_NODES:
 %   if node.is_buildable():
+%     if node.description:
```

Comment:
Should we add some default documentation here?

Reply:
I think if there’s no comment, I don’t think there’s much value in generating one since all the information the comment can show is easily visible in the source code.